File: app/src/main/java/com/example/user/eventest/eventbus/events/DatePickerUpdateEvent.java
package com.example.user.eventest.eventbus.events;

import android.util.Log;

import java.util.Calendar;

/*
 * Created by User on 01.01.2018.
 */

public class DatePickerUpdateEvent {
    private Calendar message;

    public DatePickerUpdateEvent(Calendar message) {
        this.message = message;
        String TAG = "EventBus";
        Log.d(TAG, this.getClass().getCanonicalName());
    }

    public Calendar getMessage() {
        return message;
    }
}

File: app/src/main/java/com/example/user/eventest/eventbus/events/DatePickerUpdateEvent.kt
package com.example.user.eventest.eventbus.events

import android.util.Log
import java.util.*

/**
 * Created by DR
 * on 01.01.2018.
 */

class DatePickerUpdateEvent(var message: Calendar) {
    init {
        Log.d(TAG, this.javaClass.canonicalName)
    }
}
